clearwater_cassandra/cassandra_plugin.py

```diff
@@ -246,6 +245,25 @@ def wait_for_cassandra(self):
         # scripts get run
         run_command(["sudo", "service", "clearwater-infrastructure", "restart"])
 
+    def add_cassandra_schemas(self):  # pragma: no cover
+        # Adding the schemas can fail when Cassandra has just started. If it
+        # does fail, simply try again until it succeeds.
```

Comment:
oh ok so if we fail here the node will be set up wrong, so we might as well just stick around until we succeed here?